Fix test workflow

Output when running `flutter pub get`: 

Resolving dependencies...
The current Dart SDK version is 3.2.4.

Because flutter_to_pdf requires SDK version >=3.4.0 <4.0.0, version solving failed.


You can try the following suggestion to make the pubspec resolve:
* Try using the Flutter SDK version: 3.22.0. 
Error: Process completed with exit code 1.
-------

From: https://github.com/joseph-grabinger/flutter_to_pdf/actions/runs/9164293458/job/25195279986

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -21,7 +21,7 @@ jobs:
       - name: Clone Flutter repository with master channel
         uses: subosito/flutter-action@v2
         with:
-          flutter-version: '3.16.7'
+          flutter-version: '3.22.0'
           channel: 'stable'
       - run: flutter doctor -v
       - run: flutter pub get

--- a/.gitignore
+++ b/.gitignore
@@ -30,6 +30,7 @@ migrate_working_dir/
 .dart_tool/
 .packages
 build/
+/linux/**
 
 # Coverage directory
 coverage/

